.github/workflows/run-ci-for-external-forks.yml

```
@@ -32,13 +35,15 @@ jobs:
           team: wrangler
 
       - name: Stop workflow if user is not a team member
-        if: ${{ steps.teamAffiliation.outputs.isTeamMember == false }}
+        if: ${{ steps.teamAffiliation.outputs.isTeamMember == 'false' }}
```

Would it be safer to do:

```suggestion
        if: ${{ steps.teamAffiliation.outputs.isTeamMember != 'true' }}
```